ledger/ledger-api-client/src/test/suite/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlowTest.scala

```diff
@@ -261,17 +261,7 @@ class CommandTrackerFlowTest
 
       "timeout the command when the MRT passes" in {
 
-        val Handle(submission, results, _, completionStreamMock) =
-          runCommandTrackingFlow(allSubmissionsSuccessful)
-
-        submission.sendNext(submitRequest)
-
-        completionStreamMock.send(
-          CompletionStreamElement.CheckpointElement(
-            Checkpoint(Some(fromInstant(mrt.plus(shortDuration))))))
-
-        results.expectNext(
-          Ctx(context, Completion(commandId, Some(Status(Code.ABORTED.value, "Timeout")))))
+        // TODO(RA): test timeouts
```

Comment:
Please remove empty test cases and track addition of such tests elsewhere. This gives false security as someone looking at the test results may assume that you are actually testing this piece of functionality.

Reply:
@gerolf-da https://github.com/digital-asset/daml/issues/5183

Can we somehow test timeouts? We'd have to use a short deduplication time, and then emit a second event to force the timeout, right?

Reply:
You can force the timeout in command tracker by using a super short deduplication time (e.g. 10ms). The command might actually still go through, but within the command tracker it will probably be reported as timed out.

Reply:
Wouldn't the functionality offered by Akka TestKit help here?
https://doc.akka.io/docs/akka/current/stream/stream-testkit.html#streams-testkit